styles/globals.css

```diff
@@ -21,6 +21,21 @@ code.language-unknown::before, code.language-unknown::after {
   content: ""
 }
 
+
+
+@media (width < 400px ){
+  .buggyTable{
+    width: 100%;
+    overflow: auto;
+  }
+  .fullwidth-breakwords{
+    width: 100%;
+    word-break:break-all 
+  }
```

Comment:
I think Tailwind can do this for us

Reply:
I not sure why but Tailwinds break-all doesn't seem to apply over here So for this reason I use Normal CSS.
![image](https://user-images.githubusercontent.com/52135949/215967226-ec2b6b24-3062-4849-9aa1-afa15086cfe9.png)